tbx/project_styleguide/templates/patterns/molecules/streamfield/blocks/featured_services_block.html

```diff
@@ -1,26 +1,25 @@
 {% load wagtailcore_tags wagtailimages_tags %}
-<div class="grid__featured-services {% if value.intro %}featured-services--with-intro{% endif %}">
-    {# Section heading #}
+<div class="grid__featured-services">
```

Comment:
Suggestion: I think for this to be valid BEM syntax we'd need to add the `featured-services` class here, so it'd be:
```
<div class="grid__featured-services featured-services">
```

Reply:
right! thanks for pointing this out, I've implemented the change